plc4j/karaf-features/karaf-itest/src/test/resources/etc/system.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
```

Comment:
You don't need this file, it will be created by `editConfigurationPut()` method using the Karaf original one (from the archive).